### ISSUE TITLE
Empty command or just pressing "Enter" throws error and usage help

### ISSUE THREAD

**Issue by SudarAbisheck** [awslabs/aws-shell#186]
Why does just pressing the "Enter" key throws "aws: error: too few arguments" and usage help instead of just creating anther aws> prompt? I would like to have some room between by commands but every time I press the Enter key, it just throws error. I expect the normal behaviour of any shell to happen and it annoys me when it doesn't happen.

**Comment by joguSD** [comment]
I agree that the current behavior is not very useful and just creating another prompt is probably better. Marking as an enhancement, thanks for the request.
